packages/lisk-transactions/test/utils/validation/validation.ts

```diff
@@ -369,4 +370,47 @@ describe('validation', () => {
 			return expect(isValidInteger(-6)).to.be.true;
 		});
 	});
+
+	describe('#isNullByteIncluded', () => {
+		const validStrings = [
+			'lorem ipsum',
+			'lorem\u0001 ipsum',
+			'loremU00000001 ipsum',
```

Comment:
'U00000000' should also be valid but it is not (see https://github.com/LiskHQ/lisk/commit/5c7e2944383bf369f1343fba62c71e84de6f9664)